src/views/DatasetManager/components/DatasetList/DatasetList.js

```diff
@@ -208,18 +156,12 @@ export const DatasetList = () => {
             userPermissions={userPermissionsOnDataset}
             necessaryPermissions={[ACL_PERMISSIONS.DATASET.DELETE]}
           >
-            <IconButton
-              onClick={(event) => askConfirmationToDeleteDialog(event, dataset)}
-              data-cy={`dataset-delete-button-${dataset.id}`}
-              disabled={dataset.ingestionStatus === INGESTION_STATUS.PENDING}
-            >
-              <DeleteForeverIcon />
-            </IconButton>
+            <DeleteDatasetButton dataset={dataset} location="" />
```

Comment:
Would it be possible to also move the parent component `PermissionsGate` ? The future button in the Overview will need the same behavior :slightly_smiling_face: